mlir/include/mlir/Dialect/OpenMP/OpenMPOpsInterfaces.td

```diff
@@ -0,0 +1,34 @@
+//===-- OpenMPOpsInterfaces.td - OpenMP op interfaces ------*- tablegen -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This is the OpenMP Dialect interfaces definition file.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef OpenMP_OPS_INTERFACES
+#define OpenMP_OPS_INTERFACES
+
+include "mlir/IR/OpBase.td"
+
+def OutlineableOpenMPOpInterface : OpInterface<"OutlineableOpenMPOpInterface"> {
+  let description = [{
+    OpenMP operations whose region will be outlined will implement this
+    interface. These operations will
```

Comment:
Looks like some words are missing here.

Reply:
Removing the incomplete sentence for now. Don't remember what I was planning to write.